# My scripts  
  
This repository contains my scripts from the `/usr/local/bin/my_scripts` directory.  
Most of the scripts are written in Perl in the good old days.  
  
The following is a brief description of each program (to come)  
  
GenSkewApp.jar  
N50  
N50.p  
README.md  
R_skripts  
aaRSpplacer  
aaRSprofileG  
aaRSprofileT  
abyss-pe-auto  
adaptercleaner  
addngs  
alignall  
alignall.old  
aligner  
aln2aln  
aln2aln.py  
bowtie2coverage  
cc2ps  
cc2ps_editing  
codeml.old  
count.reads  
countseq  
do4all  
extract_stem-loops  
fa2coevol  
fasta  
fasta2  
gb2gor  
gb2map  
gb2mf  
genomestats  
get-contigs  
get-contigs-org  
`gname2num.phy`, change gene names to numbers in a GO file
go2gr  
go2nex  
go2nex.old  
go2nex_out  
go2ps  
go2ps.old  
go2psv  
gorcoder  
gorcom  
gorcom.old  
gorcom_previous  
gorcom_v1  
gornex  
gorsim  
gotree  
helptext.txt  
length-hist  
make_codon_table  
mcons  
mcons.old  
mf2cc, create a coordinate file or a gene order file from a masterfile.  
mf2fasta  
mf2fasta.new  
mkngs  
`mpepper`, `pepper -t` on multiple files  
`mtann`, quick annotation of mitochondrial genomes  
`newalign`, program to select, align, and concatenated sequences from individual mitochondrial genes  
ntsoapalign  
pal2nal  
pal2nal_all  
paml  
parse_codeml  
pbdatebars  
phylip2fasta  
plate  
plotcoevoltree  
plotcoevoltree.R  
puzzleboot  
ramdisk  
read_paml_rates  
remove-small-contigs  
rename_trees  
renamegb  
rmbarcode  
rmngs  
rmtrees  
rmtrees_consel  
sauveguard  
seq2seq  
`skiptRNA`, a program to convert multiple GO files into a NEXUS file for MPMEMG analysis  
`soapalign`, replaced by `newalign`; a program to select, align, and concatenated sequences from individual mitochondrial genes  
soapalign.mod  
soapalign.pretdy  
sort-seqs  
split2cont  
`splitGO`, a perl script to split a multi-species go file into individual go files (used as input by `skiptRNA` program) 
sto2ali  
talign  
temp  
temp.ERR  
testal  
trimfastq  
updategb  
updategb2  
updatemf  
